packages/fxa-content-server/app/scripts/lib/experiments/grouping-rules/index.js

```diff
@@ -17,6 +17,7 @@ const experimentGroupingRules = [
   require('./push'),
   require('./third-party-auth'),
   require('./generalized-react-app'),
+  require('./new-recovery-key-UI'),
```

Comment:
I don't love the file capitalization here but we're going to be removing this experiment logic soon so seems fine for now.